### PR TITLE
refactor container PasswordReset to functional component

### PR DESCRIPTION
**Pull request by warisanwar**
#58 issue

### CODE REVIEW

File: client/src/containers/PasswordReset.js [hunk 1]
```diff
@@ -16,124 +16,117 @@ const queryString = require("qs"); // eslint-disable-line
...
 
     if (password !== passwordConfirm) {
-      this.setState({ error: "The passwords are not matching" });
+      setError("The passwords are not matching");
       return;
     }
-
-    this.setState({ loading: true, success: false, error: false });
+    setLoading(true);
+    setSuccess(true);
```
Comment (razvanilin) [inline client/src/containers/PasswordReset.js:45]: this should be `setSuccess(false)`

**Review by razvanilin — CHANGES_REQUESTED** [review]

File: client/src/containers/PasswordReset.js (hunk 1 above)
Comment (warisanwar) [reply client/src/containers/PasswordReset.js]: Fixed it! 💯

**Comment by warisanwar** [comment]
Changes are already done!You can have a final look and merge it.

**Review by razvanilin — APPROVED** [review]
Thanks for the changes! I'll merge this soon

**Pull request merged**